server.js

```diff
@@ -1966,6 +1966,50 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub release-download-count integration.
+camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];  // eg, qubyte/rubidium
+  var repo = match[2];
+  var id = match[3];
+  var asset_name = match[4].toLowerCase();
```

Comment:
i'm adding the comment inline with examples, any other suggestions for surfacing the examples of this value to visitors on the web page?